src/views/ManageList.jsx

```diff
@@ -23,13 +23,16 @@ export function ManageList({ listPath, currentUserId }) {
 	//Enter key also submits the form as long as user is on one of the input field
 	async function handleSubmit(e) {
 		e.preventDefault();
+
 		formData.daysUntilNextPurchase = +formData.daysUntilNextPurchase;
 		let result = await addItem(listPath, formData);
-		if (result.success) {
+		if (result && result.success) {
```

Comment:
Out of curiosity, would you mind sharing the thought process behind using the logical && in the conditional statements instead of only `result.success` or `result.error`?

Reply:
For sure--we actually initially tried it without the logical && and got an error. Adding the logical && checks that `result` isn't null or undefined before checking the properties of `result`.